.ruby-gemset

```diff
@@ -0,0 +1 @@
+heartcheck
```

Comment:
Why do you commit a gemset?  
I hate to see that people using it yet. Why do you not trust bundler?